### PR TITLE
add docker-compose.yaml for dockerhub and daocloud

### PR DESCRIPTION
**Pull request by wemeya**

### CODE REVIEW

File: contrib/docker-compose.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+choice=${choice:-1}
+if [ $choice == '2' ]
+then
+  sed -i -- 's/daocloud.io/docker.io/g' docker-compose.yml.daocloud
+elif [ $choice != '1' ]
+then
+  sed -i -- 's/daocloud.io/'$choice'/g' docker-compose.yml.daocloud
+fi
+mv docker-compose.yml docker-compose.yml.bak
+mv docker-compose.yml.daocloud docker-compose.yml
```
Comment (reasonerjt) [inline contrib/docker-compose.sh]: maybe rename the docker-compose.yml.daocloud to docker-compose.yml.tmpl?

File: contrib/README.md [hunk 2]
```diff
@@ -1 +1,3 @@
 The `contrib` directory contains documents, scripts, and other helpful things which are contributed by the community.
+
+docker-compose.sh is used to configure docker-compose.yml to pull images from platform like daocloud.io, docker hub and others. If you don't want to waste time on building images, you can excute this script to pull images from platform you like.Currently, we only support daocloud.io and docker hub.
```
Comment (reasonerjt) [inline contrib/README.md]: excute -> execute

Comment (reasonerjt) [inline contrib/README.md]: And there should be space after period. 

.....you like.  Currently.....

**Comment by wemeya** [comment]
@hainingzhang @reasonerjt 
Already update! By the way, shall we put the script and the doc in the Deploy? It won't be easy for users to find this in the current directory. That way will be more clear and easy for user to choose either build or pull.

**Comment by reasonerjt** [comment]
@wemeya could you squash the commits?

**Comment by wemeya** [comment]
@reasonerjt  ok

**Pull request closed without merging**